client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/ServerConfiguration.java

```diff
@@ -61,6 +63,11 @@ public String getUrl() {
   }
 
   @CheckForNull
+  public String getOrganization() {
```

Comment:
I would name it getOrganizationKey() (same comment for the member var and Builder methods)